tox.ini

```diff
@@ -14,10 +15,13 @@ setenv =
     PIP_INDEX_URL = https://pypi.python.org/simple/
 deps =
     py27: mock
+    py27: pyhamcrest==1.9.0
```

Comment:
Travis build failed when using pyhamcrest 1.10.0 with python 2.7. I don't fully understand why, but using pyhamcrest 1.9 fixed the build.